packages/ui-date-time-input/src/DateTimeInput/__examples__/DateTimeInput.examples.ts

```diff
@@ -41,7 +41,8 @@ export default {
         { text: 'success message', type: 'success' }
       ]
     ],
-    interaction: ['enabled', 'disabled']
+    interaction: ['enabled', 'disabled'],
+    disabledDates: [undefined, ['2018-01-18T16:00']]
```

Comment:
I would somehow filter these examples, because the disabled days cannot be seen when the dropdown is not open anyway, maybe only when the current date is invalid and the message is shown.

Reply:
true, I just set disabledDates to undefined because we're never showing the calendar popup